PythonWrapperGenerator/llvm-config-envision.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
This file looks identical to the one in `CppImport`. We should absolutely not have two files. We should put a single file in `../misc` where there are a few other scripts, and use it from both projects.